go/oasis-node/cmd/genesis/genesis.go

```diff
@@ -14,10 +14,14 @@ import (
 	flag "github.com/spf13/pflag"
```

Comment:
Might make sense to start versioning the genesis file and including a version field in the doc? (specially now as it might be changing a bit more often)